src/main/java/me/champeau/gradle/japicmp/ignore/entity/Entity.java

```diff
@@ -0,0 +1,12 @@
+package me.champeau.gradle.japicmp.ignore.entity;
+
+//TODO: Implement version supporting
+public abstract class Entity {
+  private final String version;
+
+  protected Entity(String version) {
+    this.version = version;
+  }
+
+  public abstract String getClassName();
```

Comment:
Looks like all subclasses have exactly the same implementation

Reply:
Yeap, it always calls Element.getCanonicalName, but we do not have access to element field here.

Reply:
Every subsclass is initialized with at least one final `Element`. If we add `Element` to `Entity` constructor, we won't need to duplicate the implementation of `getClassName()`